pstress/pstress_log_to_sql_converter.sh

```diff
@@ -0,0 +1,133 @@
+###############################################################################################
+#                                                                                             #
+# Author  :  Mohit Joshi                                                                      #
+# Script  :  pstress_log_to_sql_converter.sh                                                  #
+# Created :  05-May-2022                                                                      #
+# Purpose :  The script is used to process pstress thread logs to filter out                  #
+#            SQLs from it                                                                     #
+#                                                                                             #
+# Usage   :                                                                                   #
+# 1. The user can pass the path of pstress log file as shown below to convert it in SQL file  #
+# ./pstress_log_to_sql_converter.sh --logdir <val>                                            #
+#                                                                                             #
+# 2. The user can use this script to also execute the converted SQL file against a running    #
+# server by setting the path --mysql-client and --socket                                      #
+# ./pstress_log_to_sql_converter.sh --logdir <val> --mysql-client <val> --socket <val>        #
+#                                                                                             #
+# For more info:                                                                              #
+# ./pstress_run_log_to_sql_converter.sh --help                                                #
+###############################################################################################
+
+# Helper Function
+Help() {
+  echo "usage: $BASH_SOURCE --logfile <val>"
```

Comment:
where would be outfine generated? provide that option default to current directory

Reply:
It defaults to current directory